tools/tvk-oneclick/tvk-oneclick.sh

```diff
@@ -301,22 +314,29 @@ EOF
 #This module is use to install license
 install_license() {
   tvk_ns=$1
+  ret=$(kubectl get license -n "$tvk_ns" 2>> >(logit) | grep -q Active)
+  ret_code=$?
+  ret=$(kubectl get license -A 2>> >(logit) | grep -q Active)
```

Comment:
License can be created in TVK's `INSTALL_NAMESPACE` only so check in that ns only

Reply:
Done